packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/extraction/Extraction.java

```diff
@@ -2,29 +2,71 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.databind.JsonNode;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import lombok.Data;
-import lombok.experimental.Accessors;
-import lombok.extern.slf4j.Slf4j;
 import software.uncharted.terarium.hmiserver.annotations.TSModel;
 
-@Slf4j
 @Data
 @TSModel
 public class Extraction {
 
 	private String extractedBy;
 
 	// Layout infomration, not used, type these out later - Mar 20
-	private JsonNode pages;
-	private JsonNode body;
-	private JsonNode groups;
+	// private JsonNode pages;
+	private Map<String, ExtractionPage> pages;
```

Comment:
Will these require a migration?

Reply:
I don't believe so, this is typing out and should be compatible with what had been in the JsonNode.